packages/cactus-plugin-ledger-connector-corda/src/test/typescript/integration/monitor-transactions-v4.8.test.ts

```diff
@@ -0,0 +1,760 @@
+/**
+ * Test low-level state change monitoring interface in Kotlin Corda v4 component.
+ */
+
+// Contants: Log Levels
+const testLogLevel: LogLevelDesc = "debug";
+const sutLogLevel: LogLevelDesc = "info";
+
+// Contants: Test ledger
+const ledgerImageName = "ghcr.io/outsh/cactus-corda-4-8-all-in-one-obligation";
+const ledgerImageVersion = "2022-03-31-2dd619d";
+const partyARpcUsername = "user1";
+const partyARpcPassword = "password";
+const partyBRpcUsername = partyARpcUsername;
+const partyBRpcPassword = partyARpcPassword;
+const stateToMonitor = "net.corda.samples.example.states.IOUState";
+const flowToInvoke = "net.corda.samples.example.flows.ExampleFlow$Initiator";
+
+// Contants: Kotlin connector server
+const kotlinServerImageName = "ghcr.io/outsh/cactus-connector-corda-server";
```

Comment:
@petermetz Same as above

Reply:
> @petermetz Same as above

@outSH Here's the new tag, please update the code and then we are good to merge IMO
`ghcr.io/hyperledger/cactus-connector-corda-server:2022-04-18-a8a7ed1--1956`